### PR TITLE
chore(float-button):  remove unused css code

### PR DESCRIPTION
**Pull request by zbw-zbw**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      fix componentCls display props      |
| 🇨🇳 Chinese |     修复组件通用样式 display 属性      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 8e8a676</samp>

Fixed a vertical alignment bug in the `float-button` component by changing its `display` property to `flex`. Enhanced the accessibility and style of the component in `components/float-button/style/index.ts` and other files.

### <samp>🤖 Generated by Copilot at 8e8a676</samp>

* Changed the `display` property of the `sharedFloatButtonStyle` to `flex` to fix the icon alignment bug ([link](https://github.com/ant-design/ant-design/pull/44587/files?diff=unified&w=0#diff-fc14ffe666c4ac5e79188e926f924116e9e8a9c57e2717b2a490d878b09e72d9L223-R223))

### CODE REVIEW

**Comment by afc163** [comment]
解决什么问题？

**Comment by zbw-zbw** [comment]
> 解决什么问题？

其实也不算问题吧 优化代码

**Comment by afc163** [comment]
具体优化的点在于？

**Comment by zbw-zbw** [comment]
> 具体优化的点在于？

display block -> flex

因为下面使用了 flex 布局水平垂直居中

align-item: center
justify-content: center

所以我理解这里是要使用 flex 布局的

File: components/float-button/style/index.ts [hunk 1]
```diff
@@ -220,7 +220,7 @@ const sharedFloatButtonStyle: GenerateStyle<FloatButtonToken, CSSObject> = (toke
       position: 'fixed',
       cursor: 'pointer',
       zIndex: 99,
-      display: 'block',
+      display: 'flex',
       justifyContent: 'center',
       alignItems: 'center',
```
Comment (afc163) [inline components/float-button/style/index.ts:225]: 确实 justifyContent alignItems 都没啥用，直接把这三个属性都删了吧。

Comment (zbw-zbw) [reply components/float-button/style/index.ts:225]: > 确实 justifyContent alignItems 都没发挥作用，直接把这三个属性都删了吧。

好的 也可以

Comment (zbw-zbw) [reply components/float-button/style/index.ts]: > 确实 justifyContent alignItems 都没发挥作用，直接把这三个属性都删了吧。

done!

**Review by afc163 — APPROVED** [review]

File: components/float-button/style/index.ts [hunk 2]
```diff
@@ -220,9 +220,6 @@ const sharedFloatButtonStyle: GenerateStyle<FloatButtonToken, CSSObject> = (toke
       position: 'fixed',
       cursor: 'pointer',
       zIndex: 99,
-      display: 'block',
```
Comment (li-jia-nan) [inline components/float-button/style/index.ts:223]: 这个不能删除

Comment (yoyo837) [reply components/float-button/style/index.ts:223]: 我也丢个链接在这边 https://github.com/ant-design/ant-design/pull/44707

**Pull request merged**